### ISSUE TITLE
Rotating/ manipulating the components or connection nodes

### ISSUE THREAD

**Issue by Prokamp** [Rustrician/rustrician-public#15]
I believe that we should have a way to rotate the components or their connection nodes as to more effectively create a clean more organized schematic.

**Comment by robisodd** [comment]
Agreed, though a change would have to be implemented to tell which nodes are input and which output.  Perhaps adding a carat to the circle (such as "O>" or "O<" instead of the current "O") would work.  Or (and?), perhaps the wires themselves could show moving chevrons instead of moving bars (so ">>>" or "<<<" instead of the current "| | |") to indicate electricity flow (helpful for non-animated screenshots).
